tests/define_hooks_track_operations_project.py

```diff
@@ -0,0 +1,111 @@
+from define_hooks_test_project import HOOKS_ERROR_MESSAGE
+
+from flow import FlowProject
+from flow.hooks import TrackOperations
+
+
+class _HooksTrackOperations(FlowProject):
+    pass
+
+
+track_operations = TrackOperations(strict_git=False)
+
+
+"""
+Strict Git False
+"""
+
+
+@_HooksTrackOperations.operation_hooks.on_start(
+    TrackOperations(strict_git=False).on_start
+)
+@_HooksTrackOperations.operation_hooks.on_exception(
+    TrackOperations(strict_git=False).on_exception
+)
+@_HooksTrackOperations.operation_hooks.on_success(
+    TrackOperations(strict_git=False).on_success
+)
+@_HooksTrackOperations.operation
+def strict_git_false(job):
+    if job.sp.raise_exception:
+        raise RuntimeError(HOOKS_ERROR_MESSAGE)
+
+
+@_HooksTrackOperations.operation_hooks.on_start(
+    TrackOperations(strict_git=False).on_start
+)
+@_HooksTrackOperations.operation_hooks.on_exception(
+    TrackOperations(strict_git=False).on_exception
+)
+@_HooksTrackOperations.operation_hooks.on_success(
+    TrackOperations(strict_git=False).on_success
+)
+@_HooksTrackOperations.operation(cmd=True, with_job=True)
+def strict_git_false_cmd(job):
+    if job.sp.raise_exception:
+        return "exit 42"
+    else:
+        return "touch base_cmd.txt"
+
+
+@track_operations.install_operation_hooks(_HooksTrackOperations)
+@_HooksTrackOperations.operation
+def strict_git_false_install(job):
+    if job.sp.raise_exception:
+        raise RuntimeError(HOOKS_ERROR_MESSAGE)
+
+
+@track_operations.install_operation_hooks(_HooksTrackOperations)
+@_HooksTrackOperations.operation(cmd=True, with_job=True)
+def strict_git_false_install_cmd(job):
+    if job.sp.raise_exception:
+        return "exit 42"
+    else:
+        return "touch base_cmd.txt"
+
+
+"""
+Strict Git True
+"""
+
+
+try:
```

Comment:
Why is this in a `try` block? It seems like we're running a test without caring what the result is. Consider marking these tests with `xfail` or `skip`.